src/test/ui/privacy/associated-item-privacy-type-binding.rs

```diff
@@ -8,19 +8,19 @@ mod priv_trait {
     pub trait PubTr: PrivTr {}
 
     pub macro mac1() {
-        let _: Box<PubTr<AssocTy = u8>>;
+        let _: Box<dyn PubTr<AssocTy = u8>>;
```

Comment:
Accidentally committed to this PR? Or does it have something to do with suggestions?

Reply:
Probably to avoid warning output unrelated to what's being tested.